test/OpenTelemetry.Exporter.Instana.Tests/OpenTelemetry.Exporter.Instana.Tests.csproj

```diff
@@ -13,6 +14,7 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.2" Condition="'$(TargetFramework)' == 'net462'"/>
```

Comment:
Why is this being added?

Reply:
Microsoft.NET.Test.Sdk -> [Microsoft.TestPlatform.TestHost](https://www.nuget.org/packages/Microsoft.TestPlatform.TestHost/17.3.2#dependencies-body-tab) -> Newtonsoft.Json for .NET (Core). There is no such reference for .NET Framework.

At least one tests needs it https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/e4582ffece4d43254f34d081aec78dfea83073ac/test/OpenTelemetry.Exporter.Instana.Tests/InstanaSpanSerializerTests.cs.

As I added execution tests for NET Fx 4.6.2 in this PR this change is needed.